wisl/lib/semantics/wislCMemory.ml

```diff
@@ -11,6 +11,32 @@ type fix_t = unit
 
 type t = WislCHeap.t
 
+module WislLogging = Logging.Make (struct
+  type nonrec t = t
+
+  let file_reporter =
+    object (self)
+      inherit [t] Logging.FileReporter.t
+
+      method private target_lang mem =
+        Format.fprintf self#formatter "Something about memory: %s"
+        @@ WislCHeap.str mem
+    end
+
+  let database_reporter =
+    object
+      inherit [t] Logging.DatabaseReporter.t
+
+      method private serialize_target_lang = WislCHeap.str
+    end
+
+  let reporters =
+    [
+      (file_reporter :> t Logging.Reporter.t);
+      (database_reporter :> t Logging.Reporter.t);
+    ]
+end)
+
```

Comment:
For demonstrative purposes, will remove before merging.

Reply:
Ok sorry I should have reviewed that yesterday, but you have a lot of things to do today anyway
If I may add something, we need to bite a bullet here that I've tried to avoid but it seems necessary before merging this.
Can you write atd generators for the wisl memory type and serialize it into memory ?